firefox_puppeteer/ui/windows.py

```diff
@@ -60,22 +60,17 @@ def close(self, handle):
         # TODO: Maybe needs to wait as handled via an observer
         return self.marionette.close_chrome_window()
 
-    def close_all(self, exceptions=None):
+    def close_all(self, exceptions=[]):
         """Closes all open chrome windows.
 
         There is an optional `exceptions` list, which can be used to exclude
         specific chrome windows from being closed.
 
-        :param exceptions: Optional, list or a single entry of handles or
-         :class:`BaseWindow` instances not to close
+        :param exceptions: Optional, list of :class:`BaseWindow` instances
+                           not to close
```

Comment:
We do not want such an indentation. Please follow what the other code in this file is using.